packages/main-api/src/features/notification/dto/list_notifications_response.rs

```diff
@@ -0,0 +1,8 @@
+use crate::features::notification::NotificationResponse;
+use crate::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize, Default, aide::OperationIo, JsonSchema)]
+pub struct ListNotificationsResponse {
+    pub items: Vec<NotificationResponse>,
+    pub bookmark: Option<String>,
+}
```

Comment:
```suggestion
pub struct NotificationsResponse = ListItemsResponse<NotificationResponse>;
```

@blikq  utilize our common strucut.